snapd-glib/requests/snapd-get-change.c

```diff
@@ -53,12 +54,18 @@ _snapd_get_change_get_data (SnapdGetChange *self)
     return self->data;
 }
 
+void
+_snapd_get_change_set_accessories_change (SnapdGetChange *self, gboolean accessories_change)
```

Comment:
I think this might be cleaner if instead `_snapd_get_change_set_path` is done which defaults to `/v2/changes` - that removes some of the special casing of what accessories are and allows another case in the future in case this is a pattern that snapd will continue to use.

Reply:
Okay. For what it is worth, `/v2/accessories/changes` was picked over `/v2/accessories/themes/changes` in the hope that we wouldn't add a third. It doesn't make much of a difference either way on the snapd-glib side though.